components/combobox/private/menu.jsx

```diff
@@ -508,16 +509,12 @@ const Menu = (props) => {
 			{props.hasMenuSpinner && (
 				<li role="presentation" className="slds-listbox__item">
 					<div className="slds-align_absolute-center slds-p-top_medium">
-						<div
-							role="status"
-							className="slds-spinner slds-spinner_x-small slds-spinner_inline"
-						>
-							<span className="slds-assistive-text">
-								{props.assistiveText.loadingMenuItems}
-							</span>
-							<div className="slds-spinner__dot-a" />
-							<div className="slds-spinner__dot-b" />
-						</div>
+						<Spinner
```

Comment:
Can now use the inline spinner variant without container in the combobox itself.